latest/Snapshots.md

```diff
@@ -6,4 +6,8 @@ layout: default
 ## Dafny Documentation snapshots
 
 - [Current development version](https://dafny.org/dafny)
-- [Latest release snapshot](https://dafny.org)
+- [Latest release snapshot](https://dafny.org/latest)
+- [v3.9.0](https://davidcok.github.io/dafny-lang.github.io/v3.9.0)
+- [v3.9.0](https://davidcok.github.io/dafny-lang.github.io/v3.9.0)
+- [v3.9.0](https://davidcok.github.io/dafny-lang.github.io/v3.9.0)
+- [v3.9.0](https://davidcok.github.io/dafny-lang.github.io/v3.9.0)
```

Comment:
Multiple links again

Reply:
Aargh - should have found this before files were copies.  Fixed.